example/python/batch-example.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+print("""
+
+PLEASE ENSURE THAT MST IS ENABLED IN IROHA CONFIG
+
+""")
+
+from irohalib import Iroha, IrohaGrpc
+from irohalib import IrohaCrypto as ic
+
+import binascii
+
+iroha = Iroha('admin@test')
+net = IrohaGrpc()
+
+admin_private_key = open('../admin@test.priv').read()
+
+alice_private_keys = [
+    b'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506caba1',
+    b'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506caba2'
+]
+alice_public_keys = [ic.derive_public_key(x) for x in alice_private_keys]
+
+bob_private_keys = [
+    b'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506caba3',
+    b'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506caba4'
+]
+bob_public_keys = [ic.derive_public_key(x) for x in bob_private_keys]
+
+
+def send_transaction_and_print_status(transaction):
+    global net
+    hex_hash = binascii.hexlify(ic.hash(transaction))
+    print('+' * 40)
```

Comment:
Doesn't really help much without reading code itself. Consider using `print("Enter <function_name>")`

Reply:
Thanks! Added a tracer for methods' calls. Now each call will be accomplished with "Entering/Leaving X". 
Thus, it will be clear what each output means.